flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/plan/PythonCalcSplitRuleTest.scala

```diff
@@ -266,4 +266,26 @@ class PythonCalcSplitRuleTest extends TableTestBase {
 
     util.verifyTable(resultTable, expected)
   }
+
+  @Test
+  def testSplitRuleForBatch(): Unit = {
```

Comment:
I guess we can remove this test case as the split rule is applied for both the streaming and batch plans. The original tests for streaming plans are enough. What do you think?